src/styles/blocks/catalog.scss

```diff
@@ -11,7 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
+    transition: transform ease 0.3s;
```

Comment:
what about color? you also change it on hover

And move transition duration to variable and reuse it everywhere